src/components/Tasks/Toolbar.tsx

```diff
@@ -20,23 +24,66 @@ import { StoreState } from '../../store';
 
 import Menu from '../../widgets/Menu';
 
+const transitionTimeout = 500;
```

Comment:
I think the transition time is too long, should probably be 300 or maybe even less? I haven't tried modifying it, I just felt it was very slow.